modules/oe_theme_content_event/src/Plugin/ExtraField/Display/DescriptionExtraField.php

```diff
@@ -88,13 +88,16 @@ public function getLabel() {
    * {@inheritdoc}
    */
   public function viewElements(ContentEntityInterface $entity) {
+    // If we have no renderable text wee don't need a title.
+    $text = $this->getRenderableText($entity);
+    $title = !empty($text[0]['#text']) ? $this->getRenderableTitle($entity) : '';
     // Display event description using "text_featured_media" pattern.
```

Comment:
Probably we have to hide if all fields are empty. There missed Title but the image is available:
![image](https://user-images.githubusercontent.com/9432036/197772814-ce228c63-4fca-4c77-bf59-1f2f0c6707ec.png)